spec/mailers/notification_mailer_spec.rb

```diff
@@ -97,10 +103,153 @@
       let(:type) { 'new_ally_request' }
 
       subject(:email) { NotificationMailer.notification_email(recipient, data) }
+      # subject(:email) { NotificationMailer.notification_email(1, data) }
 
       it { expect(email.subject).to eq("if me | #{who_triggered_event.name} sent an ally request!") }
       it { expect(email.body.encoded).to match('<p>Please <a href="http://localhost:3000/allies">sign in</a> to accept or reject the request!</p>') }
     end
+    
+    context 'when type is new_group' do
+      let(:type) { 'new_group' }
+
+      subject(:email) { NotificationMailer.notification_email(recipient, data) }
+
+      it { expect(email.subject).to eq("if me | #{who_triggered_event.name} created a group \"#{group.name}\"") }
+      it {expect(email.body.encoded).to match("Hi #{recipient.name},")}
+      it { expect(email.body.encoded).to match("<p>#{who_triggered_event.name} created a group \"#{group.name}\":</p><p><i>#{group.description}</i></p><p>To learn more and join, #{group_link}!</p><p>Please follow our #{code_of_conduct_link}!</p>") }
+    end
+
+    context 'when type is new_group_member' do
+      let(:type) { 'new_group_member' }
+
+      it do
+        puts "data #{data} recipient #{recipient}"
```

Comment:
Should we remove this?

Reply:
This code and the line below were for debugging. I've removed both in a new commit.